common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1134,8 +1134,19 @@ def get_parent_location(self, locator, **kwargs):
 
         course = self._lookup_course(locator.course_key)
         parent_ids = self._get_parents_from_structure(BlockKey.from_usage_key(locator), course.structure)
+
+        # Possible orphan; if multiple `parent_ids` found
+        if len(parent_ids) > 1:
+            orphan_block_keys = [
+                BlockKey(id=c.block_id, type=c.block_type)
+                for c in self.get_orphans(locator.course_key)
+            ]
+            if orphan_block_keys:
```

Comment:
If both parents are non then `parents_ids` would become and empty list, and the following instruction will return `None`
```
if len(parent_ids) == 0:
    return None
```